cymetric/cycamore_root_metrics.py

```diff
@@ -0,0 +1,18 @@
+from cymetric import root_metric
+
+#root metrics that come with the archetypes within the cycamore module
+agentstate_cycamore_enrichmentinfo = root_metric(
```

Comment:
Following from the above comment on brightlite, cymetric shouldn't have a dependency on cycamore.